.github/workflows/wiki.yml

```
@@ -0,0 +1,54 @@
+      # Step 1: Check out the repository's files to access the script
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      # Step 2: Install jq, the robust JSON processor
+      - name: Install jq
+        run: sudo apt-get update && sudo apt-get install -y jq
+
+      # Step 3: Execute the wiki.sh script
+      - name: Run wiki query script
+        id: run-mcp-query
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:3:51: Double quote to prevent globbing and word splitting

(shellcheck)